presto-docs/src/main/sphinx/functions/json.rst

```diff
@@ -59,6 +59,12 @@ Cast from JSON
 
 JSON Functions
 --------------
+.. function:: is_json_scalar(json) -> boolean
+
+    Determine if ``json`` is a scalar (i.e. not an array or an object)::
```

Comment:
Let's say

```
... (i.e. a JSON number, a JSON string, ``true``, ``false`` or ``null``.
```

I am now not sure whether JSON null should be considered as a scalar, I will put discussions later.